packages/issuer-sdk-js/src/create-issuer-config.ts

```diff
@@ -48,8 +50,10 @@ export async function createIssuerConfig(params: CreateIssuerConfigParams) {
   });
 
   const signer = createKwilSigner(params.signer);
+  const secretKey = Base64Codec.decode(params.encryptionSecret);
+  const keyPair = nacl.box.keyPair.fromSecretKey(secretKey);
 
-  return { chainId, dbid, kwilClient, signer, encryptionSecret: params.encryptionSecret };
+  return { chainId, dbid, kwilClient, signer, keyPair };
```

Comment:
Please use names that indicate the key's usage.
```suggestion
  return { chainId, dbid, kwilClient, signer, encrypter };
```